app-ios/Sources/TimetableFeature/TimetableView.swift

```diff
@@ -76,10 +83,19 @@ public struct TimetableView: View {
         self.store = store
     }
 
+    static let scrollThreshold: CGFloat = 88
+
     public var body: some View {
         WithViewStore(store) { viewStore in
             NavigationView {
-                VStack(spacing: 0) {
+                ZStack(alignment: .top) {
```

Comment:
Replace VStack to ZStack for smooth expanding.

VStack|ZStack
:---:|:---:
<video src="https://user-images.githubusercontent.com/26338889/190654508-87defeae-c5bc-42c3-9018-bfed90185241.mp4" />|<video src="https://user-images.githubusercontent.com/26338889/190654739-72c86264-748a-400a-ab5a-635ada94baee.mp4" />